universum_log_collapser/e2e/static_html.test.js

```diff
@@ -1,92 +1,35 @@
+const utils = require("./utils");
+
 beforeAll(async () => {
-    const path = require('path');
-    const currDir = path.dirname(module.filename);
-    await page.goto('file://' + currDir + '/test.html');
-    await waitForHandledResults(2);
+    await utils.openPage("test.html");
+    const expFailedStepsCount = 2; // depends on test HTML content
```

Comment:
Are you sure you want to declare the exact same constant several times instead of just sharing it?

Reply:
Yes.
It's just a coincidence that 2 different HTMLs have same failed steps count. They are not related to each other in any way.
I can update one of HTMLs to have 3 failed steps to avoid embarrassment.